modules/matchPattern.js

```diff
@@ -1,21 +1,30 @@
 import pathToRegexp from 'path-to-regexp'
 
 const cache = {}
+const pathToRegexpOptions = {strict: true}
```

Comment:
Also, a small nitpick: Can you put spaces inside of the brackets for an object? It's not officially enforced, but it's kind of the standard around the code and looks good inline with everything else.